### PR TITLE
Brand footer separator changed to hide from screen readers

### PR DESCRIPTION
**Pull request by amichaelparker**
### Summary
Ref: [#280](https://github.com/cerner/terra-framework/issues/280). Moved the '/' separator out of CSS content to hide it from screen readers.

Closes [#280](https://github.com/cerner/terra-framework/issues/280)

### Additional Details
Added a theming variable the Brand Footer CSS to allow the separator to be re-colored.

### CODE REVIEW

File: packages/terra-brand-footer/src/BrandFooter.jsx [hunk 1]
```diff
@@ -136,11 +136,13 @@ const BrandFooter = ({
...
             }
             <ul className={cx('menu')}>
-              {linkGroup.links && linkGroup.links.map((link) => {
+              {linkGroup.links && linkGroup.links.map((link, index) => {
                 const spreadTarget = link.target !== undefined ? { target: link.target } : {};
+                const separator = (!isVertical && index >= 1) ? <span className={cx('separator')} aria-hidden>/</span> : '';
                 return (
                   <li className={cx('list-item')} key={link.text + link.href}>
                     <a className={cx('link')} href={link.href} {...spreadTarget}>
+                      {separator}
```
Comment (yuderekyu) [inline packages/terra-brand-footer/src/BrandFooter.jsx]: we may have to reconsider where to render the separator within html markup - it's receiving the link's focus style. perhaps outside the list item?
screen shot 2018-11-13 at 3 33 31 pm

Comment (bjankord) [reply packages/terra-brand-footer/src/BrandFooter.jsx]: What browser are you seeing this in?

Comment (amichaelparker) [reply packages/terra-brand-footer/src/BrandFooter.jsx]: I'm also seeing this, in Chrome. Moving it outside the item appears to fix it, so I'll push a change.

**Comment by bjankord** [comment]
Tested in VoiceOver on Mac and iOS as well as JAWS 2018 in IE 11, separators were not read by the screen readers. Nice work on this @amichaelparker!

**Pull request merged**